metadata-ingestion/tests/integration/hive/test_hive.py

```diff
@@ -2,12 +2,16 @@
 
 import pytest
 from click.testing import CliRunner
+from freezegun import freeze_time
 
 from datahub.entrypoints import datahub
 from tests.test_helpers import fs_helpers, mce_helpers
 from tests.test_helpers.docker_helpers import wait_for_port
 
+FROZEN_TIME = "2020-04-14 07:00:00"
 
+
+@freeze_time(FROZEN_TIME)
```

Comment:
im surprised that this one needed a freeze_time given that it already had `mock_time`

Reply:
Me too- this was definitely necessary though.